examples/src/pages/e2e/cad/DefaultCognite3DViewerTestPageV8.tsx

```diff
@@ -2,15 +2,20 @@
  * Copyright 2021 Cognite AS
  */
 
+import { Cognite3DViewer } from '@cognite/reveal';
 import React from 'react';
+import * as THREE from 'three';
 import { registerVisualTest } from '../../../visual_tests';
 
 import { Cognite3DTestViewer } from '../Cognite3DTestViewer';
 
 function DefaultCognite3DViewerTestPageV8() {
   const modelUrl = 'primitives_v8';
 
-  return <Cognite3DTestViewer modelUrls={[modelUrl]} />;
+  return <Cognite3DTestViewer modelUrls={[modelUrl]} initializeCallback={(viewer:Cognite3DViewer) => {
+    viewer.cameraManager.setCameraState({position: new THREE.Vector3(30,10,50), 
+      target: new THREE.Vector3()});
+  }} />;
```

Comment:
```suggestion
  return <Cognite3DTestViewer modelUrls={[modelUrl]} initializeCallback={(viewer:Cognite3DViewer) => {
    viewer.cameraManager.setCameraState({position: new THREE.Vector3(30,10,50), 
      target: new THREE.Vector3()});
  }} />;
```
Instead of fixing every single usage of `Cognite3DTestViewer`, just add this into the implementation of `Cognite3DTestViewer`.